uber/templates/common.js

```diff
@@ -97,18 +97,18 @@ var $undoForm = function (path, params, linkText) {
 };
 
 function showTop (message) {
```

Comment:
These are all just style changes.  I originally had written some code in this file just below these lines, but that ended up being unnecessary.